openvpn/config.go

```diff
@@ -60,6 +60,13 @@ func (c *Config) SetTLSCrypt(cryptFile string) {
 	c.AddOptions(OptionFile("tls-crypt", cryptFile))
 }
 
+func (c *Config) SetReconnectLimits() {
+	c.setParam("connect-retry-max", "2")
+	c.setParam("remap-usr1", "SIGTERM")
+	c.setFlag("single-session")
+	c.setFlag("tls-exit")
```

Comment:
Some of params doesn't seem to be used for reconnection limit, which is the purpose of this method. I.e. `tls-exit` and `remap-usr1` seems to be used for allowing to exit vpn in some graceful way.

Reply:
i use limit in broader sense, not a number, but limitation, not to reconnect.

Reply:
ant these are parameters that enforces that limitation..

Reply:
> i use limit in broader sense, not a number, but limitation, not to reconnect.

I don't quite understand - anyways, documentation for this method is missing - maybe documenting this would explain what broader sense you have?